fastai/vision/augment.py

```diff
@@ -362,13 +362,25 @@ def _grid_sample(x, coords, mode='bilinear', padding_mode='reflection', align_co
     return F.grid_sample(x, coords, mode=mode, padding_mode=padding_mode, align_corners=align_corners)
 
 # Cell
-def affine_grid(theta, size, align_corners=None):
+def affine_grid(
+    theta:Tensor, # Transformation `Tensor`
```

Comment:
```suggestion
    theta:Tensor, # A batch of Affine transformation matrices
```
This is the same as `mat` in affine_coord

Reply:
Thanks. I fix it in the new commit